index.html

```diff
@@ -594,52 +596,57 @@
 S_\mathrm{subsurface} = \mathrm{Slab}(f_\mathrm{dielectric}, V^\infty_\mathrm{subsurface}) \ .
```

Comment:
Minor tangential comment: A few lines above here, on line 593, we say "where in general the exiting ray leaves at a different surface location than the incident ray". It feels like we could probably remove the "in general" part, since mathematically there is a zero probability of exiting at the exact point where we entered. (The probability is non-zero in practice due to rounding, but that seems too academic and negligible to mention.)

Reply:
Agreed, done in https://github.com/AcademySoftwareFoundation/OpenPBR/pull/171/commits/ffcd870d3e8c69a33aa5a632527ce144f42fc048